ros2/Dockerfile

```diff
@@ -29,6 +29,7 @@ ARG DISTRO
 FROM ros:${DISTRO} AS main
 ENV DEBIAN_FRONTEND noninteractive
 ENV RTI_NC_LICENSE_ACCEPTED yes
+ENV RTI_VERSION=5.3.1
```

Comment:
No need for `=`, unless we want to turn this into an `ARG`, in which case we can use `=` to assign a default value.